examples/Example12/Example12.cpp

```diff
@@ -0,0 +1,268 @@
+  double *energyDeposit      = new double[NumPlaced];
+  ScoringPerVolume scoringPerVolume;
+  scoringPerVolume.chargedTrackLength = chargedTrackLength;
+  scoringPerVolume.energyDeposit      = energyDeposit;
+  GlobalScoring globalScoring;
+
+  Example12(particles, energy, batch, MCCindex, &scoringPerVolume, &globalScoring, NumVolumes, NumPlaced, &hepEmState);
+
+  std::cout << std::endl;
+  std::cout << std::endl;
+  double meanEnergyDeposit = globalScoring.energyDeposit / particles;
+  std::cout << "Mean energy deposit          " << (meanEnergyDeposit / copcore::units::GeV) << " GeV\n"
+            << "Mean number of gamma         " << ((double)globalScoring.numGammas / particles) << "\n"
+            << "Mean number of e-            " << ((double)globalScoring.numElectrons / particles) << "\n"
+            << "Mean number of e+            " << ((double)globalScoring.numPositrons / particles) << "\n"
+            << "Mean number of charged steps " << ((double)globalScoring.chargedSteps / particles) << "\n"
+            << "Mean number of neutral steps " << ((double)globalScoring.neutralSteps / particles) << "\n"
+            << "Mean number of hits          " << ((double)globalScoring.hits / particles) << "\n";
+  std::cout << std::endl;
+
+  // Average charged track length and energy deposit per particle.
+  for (int i = 0; i < NumPlaced; i++) {
+    chargedTrackLength[i] /= particles;
+    energyDeposit[i] /= particles;
+  }
+
+  std::cout << std::scientific;
+#ifdef VERBOSE
+  std::cout << std::endl;
+  const int id = world->id();
+  std::cout << "World (ID " << world->id() << ") Charged-TrakL " << chargedTrackLength[id] / copcore::units::mm
+            << " Energy-Dep " << energyDeposit[id] / copcore::units::MeV << " MeV" << std::endl;
+  PrintScoringPerVolume(world, &scoringPerVolume);
+#endif
+  /*
+    // Accumulate per material.
+    double energyDepGap = 0, energyDepAbsorber = 0;
+    for (int i = 2; i < NumVolumes; i += (1 + NbOfAbsorbers)) {
+      energyDepGap += energyDeposit[i + 1];
+      energyDepAbsorber += energyDeposit[i + 2];
+    }
+
+    std::cout << std::endl;
+    std::cout << " " << GapMaterial << ": " << energyDepGap / copcore::units::GeV << " GeV" << std::endl;
+    std::cout << " " << AbsorberMaterial << ": " << energyDepAbsorber / copcore::units::GeV << " GeV" << std::endl;
+    // Accumulate per layer.
+    std::cout << std::endl;
+    std::cout << "  Layer   Charged-TrakL [mm]   Energy-Dep [MeV]" << std::endl;
+    for (int i = 0; i < NbOfLayers; i++) {
+      int layerVolume        = 2 + i * (1 + NbOfAbsorbers);
+      double chargedTrackLen = chargedTrackLength[layerVolume + 1] + chargedTrackLength[layerVolume + 2];
+      double energyDep       = energyDeposit[layerVolume + 1] + energyDeposit[layerVolume + 2];
+      std::cout << std::setw(5) << i << std::setw(20) << chargedTrackLen / copcore::units::mm << std::setw(20)
+                << energyDep / copcore::units::MeV << std::endl;
+    }
+  */
```

Comment:
remove if not used (I still think that accumulating per material would be good for comparison with Geant4)